stf/test/saros/stf/test/stf/editor/EditorByAliceTest.java

```diff
@@ -119,21 +117,27 @@ public void autoComplateProposal() throws RemoteException {
         .open();
     ALICE.remoteBot().editor(Constants.CLS1_SUFFIX).waitUntilIsActive();
     ALICE.remoteBot().editor(Constants.CLS1_SUFFIX).navigateTo(3, 0);
-    assertDoesNotContain(
-        "public static void main",
-        ALICE.remoteBot().editor(Constants.CLS1_SUFFIX).getTextOnLine(3));
+    assertFalse(
+        ALICE
+            .remoteBot()
+            .editor(Constants.CLS1_SUFFIX)
+            .getTextOnLine(3)
+            .contains("public static void main"));
 
     ALICE
         .remoteBot()
         .editor(Constants.CLS1_SUFFIX)
         .autoCompleteProposal("main", "main - main method");
-    assertContains(
-        "public static void main",
-        ALICE.remoteBot().editor(Constants.CLS1_SUFFIX).getTextOnLine(3));
+    assertTrue(
```

Comment:
Did the former fail of the assertion printed something like ("xyz was not found in abc") ? If yes then this is a drawback as it swallow the content of the content to check.